src/Components/Lobby.tsx

```diff
@@ -74,7 +75,10 @@ const Lobby = () => {
 
     // if found
     if (roomFound && !roomFound.gameStarted && roomFound.players.length <= 8) {
-      const {player, dbPlayer}: PlayerFactoryType = PlayerFactory(playerName, roomFound.players.length);
+      // add player Number
+      let newPlayerNo = Math.max(...roomFound.players.map( obj => obj.number));
```

Comment:
FYI Kris, i think this wont quite solve it. as you can run into this issue below
Where player number expands beyond 8

I reckon we may as well refactor the playerFactory code.
Ive got a solution, that ill run you through later

<img width="339" alt="Screen Shot 2023-03-01 at 10 57 11 am" src="https://user-images.githubusercontent.com/51236363/222010512-f1d51fb0-3206-461c-bc0e-ab7b5747b391.png">
<img width="323" alt="Screen Shot 2023-03-01 at 10 57 21 am" src="https://user-images.githubusercontent.com/51236363/222010517-5e1aa3e7-d9f5-44ea-af3f-81a40f181665.png">

Reply:
hmmm i see... i guess now it depends if we want to reshuffle all playerNumbers when a player leaves or allow for playerNumber to be any number. 

at least the array of players does not exceed 8 players